server/apps/postgres-app/sample-configuration/postgres.properties

```diff
@@ -10,13 +10,18 @@ database.schema=public
 # Boolean. Optional, default to false. Whether to enable row level security.
 row.level.security.enabled=true
 
+
+# ========================================= Pool Connection ==========================================
 # Integer. Optional, default to 10. The initial number of connections that are created when the pool is started.
-pool.initial.size=10
+pool.initialSize=10
 
 # Integer. Optional, default to 10. The maximum number of connections that can be created in the pool.
-pool.max.size=50
+pool.maxSize=10
 
 # Integer. Optional, default to 1800 (seconds).
 # Maximum idle time of the connection in the pool. Negative values indicate no timeout
+pool.maxIdleTime=1800
+
+# Without this property, the connection pool also support the following properties:
+# pool.acquireRetry, pool.minIdle, pool.maxLifeTime, pool.maxAcquireTime, pool.maxCreateConnectionTime, pool.maxValidationTime, pool.poolName
```

Comment:
IMO we should list the property + default value explicitly here (a reference to another link takes time to follow). e.g.

```
pool.acquireRetry
Number of retries if the first connection acquisition attempt fails. Defaults to 1.
```

Reply:
The copy-paste is easy for me, but:
- r2dbc pool has a lot of properties, but the copy-paste of all documents I think it makes noise. 
- The properties in r2dbc, we just forward in James, not modified more
About properties that not sure all understand, I think the reference to the original document is better